src/main/main.ts

```diff
@@ -223,7 +223,12 @@ const handleSlippiURIAsync = async (aUrl: string) => {
 
       const fileAlreadyExists = await fileExists(destination);
       if (!fileAlreadyExists) {
-        const dlUrl = `https://storage.googleapis.com/slippi.appspot.com/${replayPath}`;
+        let dlUrl;
+        if (replayPath.startsWith("http")) {
+          dlUrl = replayPath;
+        } else {
+          dlUrl = `https://storage.googleapis.com/slippi.appspot.com/${replayPath}`;
+        }
```

Comment:
```suggestion
        const dlUrl = (replayPath.startsWith("http")) ? replayPath : `https://storage.googleapis.com/slippi.appspot.com/${replayPath}`;
```
Can we maybe just use a ternary operator here?